gen/OpenIddict.Client.WebIntegration.Generators/OpenIddictClientWebIntegrationGenerator.cs

```diff
@@ -906,6 +909,11 @@ public sealed partial class OpenIddictClientWebIntegrationConfiguration
             {
                 settings.{{ setting.property_name }} = new Uri(""{{ setting.default_value }}"", UriKind.RelativeOrAbsolute);
             }
+            {{~ else if setting.type == 'Boolean' ~}}
+            if (!settings.{{ setting.property_name }}.HasValue && bool.TryParse(""{{ setting.default_value }}"", out var value))
+            {
+                settings.{{ setting.property_name }} = value;
+            }
```

Comment:
Using `TryParse` could potentially hide a bug if we somehow set a non-true/false value. Using `Parse()` here would immediately throw an exception.

```suggestion
            {{~ else if setting.type == 'Boolean' ~}}
            if (!settings.{{ setting.property_name }}.HasValue)
            {
                settings.{{ setting.property_name }} = bool.Parse(""{{ setting.default_value }}"");
            }
```

Reply:
Actually, I'm being stupid... there's a much better approach 🤣 

```suggestion
            {{~ else if setting.type == 'Boolean' ~}}
            if (settings.{{ setting.property_name }} is null)
            {
                settings.{{ setting.property_name }} = setting.default_value;
            }
````